src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+
+    private final List<Integer> lotto;
```

Comment:
```suggestion
    private final List<LottoNumber> lotto;
```
로또 숫자를 Integer원시값을 사용하는 것보다 `LottoNumber`를 만들어 포장해보는 것은 어떨까요?
참고
https://tecoble.techcourse.co.kr/post/2020-05-29-wrap-primitive-type/